libs/blocks/modal/modal.js

```diff
@@ -245,5 +250,8 @@ window.addEventListener('hashchange', (e) => {
   } else {
     const details = findDetails(window.location.hash, null);
     if (details) getModal(details);
+    if (e.oldURL?.includes('#search=')) {
```

Comment:
Milo should not be opinionated about the deep link values. So I don't expect to see `search=`.

Also, when I change the deeplink values by clicking on the category filters, they are not captured.